Source/WebCore/platform/win/cairo/DragImageWinCairo.cpp

```diff
@@ -327,7 +327,7 @@ DragImageRef createDragImageForLink(Element&, URL& url, const String& inLabel, T
     WebCoreDrawDoubledTextAtPoint(context, label, textPos, labelFont, topColor, bottomColor);
 
     deallocContext(contextRef);
-    return image.leak();
+    return  { image.leak(), nullprt };
```

Comment:
Typo here
```suggestion
    return  { image.leak(), nullptr };
```

Reply:
(and there's an extra space before the { I think)